### PR TITLE
main/py-requests: upgrade to 2.16.5

### PR DESCRIPTION
**Pull request by fabaff**

### CODE REVIEW

File: main/py-requests/APKBUILD [hunk 1]
```diff
@@ -2,12 +2,13 @@
...
 _pkgname=requests
-pkgver=2.13.0
+pkgver=2.16.5
 pkgrel=0
 pkgdesc="A HTTP request library for Python"
 url="http://www.python-requests.org/"
 arch="noarch"
-license="ASL-2.0"
+license="ASL 2.0"
+options="!check"
```
Comment (Shizmob) [inline main/py-requests/APKBUILD]: No upstream testsuite? If so, please add a comment indicating it.

Comment (Shizmob) [inline main/py-requests/APKBUILD]: The same comment as in #1588 for the license applies.

Comment (fabaff) [inline main/py-requests/APKBUILD]: Here the license matches the current [convention](https://wiki.alpinelinux.org/wiki/Creating_an_Alpine_package#license).

Comment added

**Comment by Shizmob** [comment]
👍 LGTM now.

**Pull request closed without merging**